trippy-config-sample.toml

```diff
@@ -170,7 +170,7 @@ tos = 0
 #
 # Extension objects with an unknown class will be parsed to capture generic information including the class, subtype,
 # length and payload bytes.
-icmp_extensions = false
```

Comment:
Add unit test to check sample config file can be deserialised and config matches default values